test/document_stores/test_opensearch.py

```diff
@@ -0,0 +1,7 @@
+import pytest
+
+from haystack.document_stores import OpenSearchDocumentStore
+
+
+def test_init_opensearch_client():
+    OpenSearchDocumentStore(index="test_index", port=9201)
```

Comment:
I plan to add several tests as I proceed introducing `opensearch-py`, so I expect this module to grow pretty fast - the idea is to let it grow organically leveraging the state-of-the-art tools and style we're introducing with the [CI refactoring](https://github.com/deepset-ai/haystack/issues/1794) instead of adding the tests in the old module and move them all at the end. Do you have a sense of the performance penalty we'd have while `test_opensearch` is almost empty, to better evaluate the tradeoff?

Reply:
Ok, as you can see from the test log it occupied a runner for about 5 mins while the actual testing took only 10 seconds. If more tests are coming soon, we should be fine with it. Currently it's just keeping the runner "busy". As it will run for any push it might slow down our CIs a bit, but not dramatically. We should have 20 runners in parallel, which means 20 hours runner time per hour. This will cost us 5 minutes runner time per push (+ additional 5 minutes when merging to master because of the windows CI). 
I'm happy to approve if we're ok with that.

Reply:
Sure, I approved